torch/onnx/symbolic_opset17.py

```diff
@@ -47,6 +47,16 @@ def layer_norm(
     # layer_norm normalizes on the last D dimensions,
     # where D is the size of normalized_shape
     axis = -len(normalized_shape)
+    scalar_type = _type_utils.JitScalarType.from_value(input)
+    dtype = (
+        _type_utils.JitScalarType.FLOAT if scalar_type is None else scalar_type.dtype()
+    )
```

Comment:
```suggestion
    scalar_type = _type_utils.JitScalarType.from_value(input, _type_utils.JitScalarType.FLOAT)
    dtype = scalar_type.dtype()
```
Would assume `FLOAT` the correct value? what if this is executed under an Autocast region? couldn't the right type would be float16 or bfloat16?